tests/drivers/touch_dev_gestures/main.c

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2020 Inria
+ *               2023 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Generic touch device test application
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include "mutex.h"
+#include "ztimer.h"
+
+#include "touch_dev.h"
+#include "touch_dev_gestures.h"
+
+#if IS_USED(MODULE_STMPE811)
+#include "stmpe811.h"
+#include "test_utils/expect.h"
+#endif
+
+#ifndef TOUCH_DEV_POLLING_PERIOD
+#define TOUCH_DEV_POLLING_PERIOD     50
+#endif
+
+#if !IS_ACTIVE(TOUCH_DEV_POLLING_MODE)
+static void _touch_event_cb(void *arg)
+{
+    mutex_unlock(arg);
+}
+#endif
+
+int main(void)
+{
+    /* Use the first screen */
+    touch_dev_reg_t *touch_reg = touch_dev_reg_find_screen(0);
+    if (!touch_reg) {
+        puts("No screen found!");
+        return -1;
+    }
+
+    touch_dev_t *dev = touch_reg->dev;
+    touch_dev_gesture_ctx_t ctx;
+
+#if IS_USED(MODULE_STMPE811)
+    uint16_t xmax = touch_dev_width(dev);
+    uint16_t ymax = touch_dev_height(dev);
+
+    stmpe811_t *stmpe811 = (stmpe811_t *)dev;
+    expect(xmax == stmpe811->params.xmax);
+    expect(ymax == stmpe811->params.ymax);
+#endif
```

Comment:
I don't think this is needed here as it's already in `tests/drivers/touch_dev` test application. Same for the stmpe811 includes above.

```suggestion
```